sysutils/puppet-agent/src/opnsense/mvc/app/models/OPNsense/PuppetAgent/PuppetAgent.xml

```diff
@@ -0,0 +1,21 @@
+<model>
+    <mount>//OPNsense/puppetagent</mount>
+    <description>
+        Manage Puppet Agent service
+    </description>
+    <items>
+        <!-- container -->
+        <general>
+            <!-- fields -->
+            <Enabled type="BooleanField">
+                <default>0</default>
+                <Required>Y</Required>
+            </Enabled>
+            <FQDN type="TextField">
+                <Required>Y</Required>
+            </FQDN>
+            <Environment type="TextField">
```

Comment:
Which value belongs in there? A static name? Maybe limit the number of allowed characters?

Reply:
Yes, a static name belongs in here. I've limited to number of allowed characters to 100 and set the Required tag to Y.